config/routes.rb

```diff
@@ -1,2 +1,7 @@
 Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      resources :joke_getter, only: [:index]
```

Comment:
change this to `:jokes` the `_getter` name is as un-REESTful as it gets.

Reply:
Ok will change it!

Reply:
Done